workspaces/topology/plugins/topology-common/package.json

```diff
@@ -13,7 +13,7 @@
   },
   "backstage": {
     "role": "common-library",
-    "supported-versions": "1.28.4",
+    "supported-versions": "1.32.2",
```

Comment:
what is this value used for? Does it mean the plugin is only compatible with Backstage 1.32.2?

Reply:
Good point

@ciiay, have we made any changes that make it incompatible with Backstage 1.28-1.31, except updating the dependencies?

I guess that we are not 100% sure at the moment. The question is valid, what kind of support does we mean here in the community plugin level.

(Can we drop that attribute for now (and keep it just in our RHDH "wrappers"s?)

@BethGriggs @vinzscam For me personal it's a big question how we can ensure that one plugin is compatible with multiple backstage versions. It's more a general community plugin question then an issue for our product.

Reply:
The major change we made was migrating from MUI v4 to MUI v5 in the topology plugin. Need to test to confirm if it makes it incompatible with Backstage 1.28-1.31.

Reply:
I have verified that the latest topology plugin works with Backstage 1.28.4.